src/site/content/en/lighthouse-best-practices/appcache-manifest/index.md

```diff
@@ -33,23 +32,27 @@ this markup in your document causes the audit to fail:
 
 {% include 'content/lighthouse-best-practices/scoring.njk' %}
 
-## Use the Cache API instead of AppCache
+## Use the Cache API instead of the Application Cache
 
 To pass this audit,
-remove the manifest from your document,
+remove the manifest from your page,
 and use the
-[service worker Cache API](https://developer.mozilla.org/en-US/docs/Web/API/Cache) instead.
+[Cache API](https://developer.mozilla.org/en-US/docs/Web/API/Cache)
+via a [service worker](https://developers.google.com/web/fundamentals/primers/service-workers/)
+instead.
 
-To help migrate from AppCache to service workers,
+To migrate from the Application Cache to service workers,
 consider using the
 [sw-appcache-behavior library](https://github.com/GoogleChrome/sw-appcache-behavior).
 This library generates a service-worker-based implementation of the behavior
-defined in an AppCache manifest.
+defined in an Application Cache manifest.
 
-See the [URL Responds With a 200 When Offline](/works-offline) audit
-reference for more resources on using service workers to make your site work
+See the [Current page responds with a 200 when offline](/works-offline) post
```

Comment:
```suggestion
See the [Current page does not respond with a 200 when offline](/works-offline) post
```

As discussed in #1384 the doc title should match the audit's `failureTitle`

Reply:
Good catch. I've submitted a feature request to slugify internal links so we don't have to manually update link text to avoid oversights like this: #1495